Command line tools - fix redirect (#10558)
mdn/content: files/en-us/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md

## A slightly more complex example

Let’s go through something a bit more complicated.

We will first try to fetch the contents of MDN's "fetch" page using the `curl` command (which can be used to request content from URLs), from `https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch`.
Try it now:

```bash
curl https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch
```

> This doesn't work because the page has been redirected (to [/Web/API/fetch](/en-US/docs/Web/API/fetch))

Reviewer: I think it would really help to still tell the reader what to expect here. It doesn't work, but I'm not sure how the reader is supposed to know that. I'd like to see a return of some text like "you won't get an output".

Author: ```suggestion
You won't get an output because the page has been redirected (to [/Web/API/fetch](/en-US/docs/Web/API/fetch)).
```
